README.md

```diff
@@ -174,6 +174,53 @@ kubectl api-resources
 
 Check out our [Contributing](CONTRIBUTING.md) and [Developer](docs/developers) guides.
 
+## Using Workspaces
+
```

Comment:
This is almost the first mention of workspaces. Fine to merge this as is; even better to outline what workspaces _are_ first.

Reply:
I agree.  I did not attempt that because I am not sure I know enough to do a good job.  How about this: after this PR merges, I will open another one to add an explanation of what workspaces are, with a draft that others can correct?